pkg/lang/javascript/aws_runtime/_keyvalue.ts

```diff
@@ -51,13 +53,6 @@ const KVStore = new Entity(
     { table: '{{.AppName}}', client: docClient }
 )
 
-import { fs as cloudFS } from './fs'
```

Comment:
unrelated cleanup, I assume?

Reply:
Yeah, I figured while I was in the neighborhood I'd leave it cleaner than when I got here :)